src/ontology/cl-edit.owl

```diff
@@ -23989,15 +24009,16 @@ SubClassOf(obo:CL_0011007 obo:CL_0000222)
 
 # Class: obo:CL_0011008 (embryonic hemocyte)
```

Comment:
Was this term meant to be obsoleted?

Reply:
That was my intention indeed, but the child class [embryonic crystal cell](http://purl.obolibrary.org/obo/CL_0000715) is used in the logical definition of GO’s [embryonic crystal cell differentiation](http://purl.obolibrary.org/obo/GO_0035165), so we need the child class around to avoid breaking GO. And if we keep the child class, it’s better to keep the parental class (along with the sibling [embryonic plasmatocyte](http://purl.obolibrary.org/obo/CL_0011009) than to have a wandering “embryonic crystal cell”.